src/Polly.Core/Simmy/Outcomes/OutcomeChaosStrategy.cs

```diff
@@ -8,65 +8,39 @@ internal class OutcomeChaosStrategy<T> : MonkeyStrategy<T>
 {
     private readonly ResilienceStrategyTelemetry _telemetry;
 
-    public OutcomeChaosStrategy(FaultStrategyOptions options, ResilienceStrategyTelemetry telemetry)
-        : base(options)
-    {
-        if (options.Fault is null && options.FaultGenerator is null)
-        {
-            throw new InvalidOperationException("Either Fault or FaultGenerator is required.");
-        }
-
-        _telemetry = telemetry;
-        Fault = options.Fault;
-        OnFaultInjected = options.OnFaultInjected;
-        FaultGenerator = options.FaultGenerator is not null ? options.FaultGenerator : (_) => new(options.Fault);
-    }
-
     public OutcomeChaosStrategy(OutcomeStrategyOptions<T> options, ResilienceStrategyTelemetry telemetry)
         : base(options)
     {
-        if (options.Outcome is null && options.OutcomeGenerator is null)
+        if (options.OutcomeGenerator is null)
```

Comment:
Make the property required and this check won't be necessary anymore.

Reply:
Good catch, it was actually already required.